exporters/writers/base_writer.py

```diff
@@ -19,22 +20,15 @@ class InconsistentWriteState(Exception):
     """
 
 
-ITEMS_PER_BUFFER_WRITE = 500000
-# Setting a default limit of 4Gb per file
-SIZE_PER_BUFFER_WRITE = 4000000000
-
-
 class BaseWriter(BasePipelineItem):
     """
     This module receives a batch and writes it where needed.
     """
     supported_options = {
-        'items_per_buffer_write': {'type': six.integer_types, 'default': ITEMS_PER_BUFFER_WRITE},
-        'size_per_buffer_write': {'type': six.integer_types, 'default': SIZE_PER_BUFFER_WRITE},
```

Comment:
we can't remove these options from writer. We will break compatibility with previous configuration files.